yard-api/src/main/java/org/kie/yard/api/model/DecisionLogic.java

```diff
@@ -27,7 +27,8 @@
         key = "type",
         value = {
                 @YamlSubtype(alias = "DecisionTable", type = DecisionTable.class),
-                @YamlSubtype(alias = "LiteralExpression", type = LiteralExpression.class)
+                @YamlSubtype(alias = "LiteralExpression", type = LiteralExpression.class),
+                @YamlSubtype(alias = "Rules", type = RuleExpression.class)
```

Comment:
If I understand correctly this is putting a whole Drools like syntax dialect inside the current AST of YaRD, which I'm not necessarily against but it seems very different to what we have now. 

Currently there is no documentation nor specification of what YaRD should be so this seems like a risky change